utils/transaction.js

```diff
@@ -1,7 +1,7 @@
 import moment from "moment";
 
 const formatDateTime = (date) => {
-  return moment(date).utc().format("YYYY-MM-DD HH:mm:ss");
+  return moment(date).format("YYYY-MM-DD HH:mm:ss");
```

Comment:
we have a function to show date and time
`import {fullDateAndTime } from '../utils/format'`
it shows the date and time in locally for user format

Reply:
`fullDateAndTime` returns `Invalid Date` for `'2023-12-13T13:42:21.000Z'`. I'll localize timestamps a bit later. Let's keep as is for now.

Reply:
it expects the Unix timestamp I guess, so we rather update what endpoint returns from the backend, as we usually return timestamps in /v2/ 

Or as an option we can update the fullDateAndTime to accept the '2023-12-13T13:42:21.000Z' format too. 

You can leave it for now, but would be good to leave a comment so I won't forget to update later :)